apps/frontend/src/pages/auth/components/sign-up-form/sign-up-form.tsx

```diff
@@ -58,21 +58,42 @@ const SignUpForm: React.FC<Properties> = ({ onSubmit }: Properties) => {
 						errors={errors}
 						label="Email"
 						name="email"
-						placeholder="email"
+						placeholder="name@gmail.com"
 						type="email"
 					/>
 					<Input
 						control={control}
 						errors={errors}
 						label="Password"
 						name="password"
-						placeholder="password"
+						placeholder="*******"
 						type="password"
 					/>
-				</div>
-				<Button label="CREATE AN ACCOUNT" type="submit" variant="dark" />
-			</form>
-		</section>
+
+					<Button label="CREATE AN ACCOUNT" type="submit" variant="dark" />
+				</form>
+			</div>
+
+			<div className={styles["logo-container"]}>
+				<span className={styles["logo"]}>LOGO</span>
+			</div>
+
+			<img
+				alt="ripple-effect-bg"
```

Comment:
alt should have a meaningful string isnisde of them, like "ripple visual effect"